ext/storage_sources/s3_store/s3_storage_source.cpp

```diff
@@ -218,13 +228,21 @@ int
 wiredtiger_extension_init(WT_CONNECTION *connection, WT_CONFIG_ARG *config)
 {
     S3_STORAGE *s3;
-    int ret;
+    S3_FILE_SYSTEM *fs;
+    WT_CONFIG_ITEM v;
 
     if ((s3 = (S3_STORAGE *)calloc(1, sizeof(S3_STORAGE))) == NULL)
         return (errno);
 
     s3->wtApi = connection->get_extension_api(connection);
-    UNUSED(config);
+
+    int ret = s3->wtApi->config_get(s3->wtApi, NULL, config, "aws_verbose", &v);
```

Comment:
This is a WiredTiger verbosity being passed into the config, not an AWS log level. Maybe just call the config key `verbose` for now?